src/common/logging.hpp

```diff
@@ -77,33 +74,35 @@ void otbrLogEnableSyslog(bool aEnabled);
  * @param[in]   aPrintStderr    Whether to log to stderr.
  *
  */
-void otbrLogInit(const char *aIdent, int aLevel, bool aPrintStderr);
+void otbrLogInit(const char *aIdent, otbrLogLevel aLevel, bool aPrintStderr);
 
 /**
  * This function log at level @p aLevel.
  *
- * @param[in]   aLevel  Log level of the logger.
- * @param[in]   aFormat Format string as in printf.
+ * @param[in]   aLevel         Log level of the logger.
+ * @param[in]   aRegionPrefix  Log region prefix.
+ * @param[in]   aFormat        Format string as in printf.
  *
  */
-void otbrLog(int aLevel, const char *aFormat, ...);
+void otbrLog(otbrLogLevel aLevel, const char *aRegionPrefix, const char *aFormat, ...);
```

Comment:
```suggestion
void otbrLog(otbrLogLevel aLevel, otbrLogRegion aRegion, const char *aFormat, ...);
```

Pass the log region instead of the prefix string?

Reply:
The `otLogRegion` is an enumeration number in OpenThread. The parameter `aRegionPrefix` is a string. So I think the type name `otbrLogRegionPrefix` is better than `otbrLogRegion`. Thought?